### ISSUE TITLE
🐛 doesn't show errors for closed file

### ISSUE THREAD

**Issue by saniul1** [biomejs/biome-vscode#712]
### VS Code version

Version: 1.96.4 (I'm using codium)

### Extension version

3.1.0

### Biome version

2.0.6

### Operating system

- [ ] Windows
- [x] macOS
- [ ] Linux

### Description

When editing (removing) code in a file, file that depends on that code should be highlighted in file tree (like eslint does). I'm catching those only when I try to build, which is a headache.

### Steps to reproduce

just editing a file that have code other files depends on (while the other files are not opened to edit)

### Expected behavior

should highlight (in file tree) all the files that have errors in them

### Does this issue occur when using the CLI directly?

Not sure / Not applicable

### Logs

```plain text

```

**Comment by nhedger** [comment]
There appears to be a misunderstanding of how Biome works and its goals.

Biome isn't responsible for reporting syntax errors and broken code, which is the responsibility of your build system.

What Biome does is format your code and provide diagnostics about recommended actions to take on your code. Additionally, Biome currently only reports diagnostics for files that are open.

**Comment by saniul1** [comment]
@nhedger I apologize for the misunderstanding. I meant biome lint errors. Is there a plan to support project-wide lint report diagnostics?

**Comment by nhedger** [comment]
There aren't any plans at the moment. We've had at least one discussion open about that: https://github.com/biomejs/biome-vscode/discussions/121
